manifest.yaml

```diff
@@ -1,7 +1,7 @@
 id: nextcloud 
 title: Nextcloud
-version: 24.0.4.3
-release-notes: Initial version for EmbassyOS
+version: 24.0.4.5
```

Comment:
Since this is the initial release, we do not need the emver component. Version should just reflect the upstream version that is being released.

Reply:
changed to 24.0.4. @kn0wmad let me know how you changed to 24.0.8, I can put that change in the PR

Reply:
code pushed